tests/fediverse/create/deliver_to_inbox.py

```diff
@@ -19,19 +19,22 @@ def __init__(self,
        self.post_content = "Good morning!"
 
 
+    @step
```

Comment:
It's good to add these. Note that the order of steps is not currently preserved. See: https://github.com/fediverse-devnet/feditest/issues/107